samples/nrf9160/at_monitor/prj.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_NRF_MODEM_LIB=y
+CONFIG_NRF_MODEM_LIB_SYS_INIT=n
+
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
```

Comment:
Does the sample actually needs to enable socket interface? It doesn't make any use of it.

Reply:
That's right 🤯 I tried to disable those but `nrf91_sockets.c` wouldn't compile without. I changed it so that now it compiles conditionally on `CONFIG_NET_SOCKETS`, and removed these entries from the sample :)